http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -113,31 +121,41 @@ abstract class LeoRoutes(
                           .map { clusterDetails =>
                             StatusCodes.OK -> clusterDetails
                           }
+                          .unsafeToFuture()
                       }
                     } ~
                     delete {
                       complete {
-                        leonardoService.deleteCluster(userInfo, GoogleProject(googleProject), clusterName).map { _ =>
-                          StatusCodes.Accepted
-                        }
+                        leonardoService
+                          .deleteCluster(userInfo, GoogleProject(googleProject), clusterName)
+                          .map { _ =>
+                            StatusCodes.Accepted
+                          }
+                          .unsafeToFuture()
```

Comment:
shall we add something like https://github.com/broadinstitute/sam/blob/79417ba82faf5030ac98847940771fd941a801fb/src/main/scala/org/broadinstitute/dsde/workbench/sam/api/api.scala to get rid of those explicit `unsafeToFuture` calls? I don't feel strongly either way..but just fyi

Reply:
Ah cool.